cppa_slack_tracker/services.py

```
@@ -0,0 +1,342 @@
+            text += f"\nComment: {comment.get('comment', '')}"
+    elif subtype:
+        text = _message_text_for_subtype(slack_message, subtype) or ""
+    else:
+        text = slack_message.get("text", "")
+
+    if user is None:
+        user_id = slack_message.get("user")
+        if not user_id:
+            if slack_message.get("text") == "A file was commented on":
+                return None
+            raise ValueError("User not found")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)